### DISCUSSION TITLE
Deque[Model] becomes List[Model] instead of List[Dict] with `jsonable_encoder`

### DISCUSSION THREAD

**Discussion by cranium** [fastapi/fastapi#9457] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
class Model(BaseModel):
    test: str

dq = deque([Model(test="test")])

jsonable_encoder(dq)  # => [Model(test="test")]

# I would expect [{"test": "test"}]
```

### Description

When passing a deque (and some other iterable collection types to jsonable_encoder), pydantic models come out of the output.

This is due to https://github.com/tiangolo/fastapi/blob/99d8470a8e1cf76da8c5274e4e372630efc95736/fastapi/encoders.py#L127 not including deque (or other iterable types).

I made a PR to address this issue: https://github.com/tiangolo/fastapi/pull/9433/files
Should other collection types be covered?

A simple workaround is to use `list()`, but that can be burdensome for some more arbitrary types.

### Operating System

Linux, Windows, macOS

### FastAPI Version

0.95.1

### Python Version

Python 3.10.6

**Comment by cranium** [comment, marked as answer]
Bumping this- any thoughts on the PR?

**Reply by tiangolo** [reply]
Thanks for the PR here! https://github.com/tiangolo/fastapi/pull/9433

I'll mark this as the answer to your question.
